packages/create-sitecore-jss/src/templates/nextjs/src/lib/next-config/plugins/cors-header.js

```diff
@@ -0,0 +1,19 @@
+﻿const config = require('../../../temp/config');
+
+/**
+ * @param {import('next').NextConfig} nextConfig
+ */
+const corsHeaderPlugin = (nextConfig = {}) => {
+  return Object.assign({}, nextConfig, {
+    async headers() {
+      return [
+        {
+          source: '/_next/:path*',
+          headers: [{ key: 'Access-Control-Allow-Origin', value: config.sitecoreApiHost }],
```

Comment:
In a production deployment scenario (e.g. on Vercel), sitecoreApiHost is not typically defined. We should probably only add the header if this is defined?